packages/app/src/device-info/presenters/sim-card.presenter.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { DeviceInfo } from "App/device/types/mudita-os"
+import { SIM } from "App/device/constants"
+import { SimCard } from "App/__deprecated__/renderer/models/basic-info/basic-info.typings"
+
+export class SimCardPresenter {
+  static toDto(data: DeviceInfo): SimCard {
+    return {
+      slot: data.selectedSim === SIM.One ? 1 : 2,
+      active: true,
+      number: 12345678,
```

Comment:
Is it ok that this value is hardcoded?

Reply:
Yep, for some reasons device info doesn't returned real phone number